lms/templates/user_dropdown.html

```diff
@@ -13,7 +13,7 @@
 <%
 ## This template should not use the target student's details when masquerading, see TNL-4895
 self.real_user = getattr(user, 'real_user', user)
-username = get_enterprise_learner_generic_name(request) or self.real_user.username
+username = get_enterprise_learner_generic_name(request) or self.real_user.profile.name
```

Comment:
@imankulov IF this is related to the header and not the discussion it should be user first name not full name

Reply:
Great catch, thanks! Addressed in 4f5907c12b3c8c48dcee9fb23ee761e8aaeeae15